lib/wax/metadata.ex

```diff
@@ -199,21 +199,14 @@ defmodule Wax.Metadata do
   def update_metadata(serial_number) do
     Logger.info("Starting FIDO metadata update process")
 
-    if Application.get_env(:wax_, :metadata_access_token) do
-      case http_get("https://mds2.fidoalliance.org/") do
-        {:ok, jws_toc} ->
-          process_metadata_toc(jws_toc, serial_number)
+    case http_get("https://mds.fidoalliance.org/") do
```

Comment:
To avoid download, we can use the `"Last-Modified"` and `"if-modified-since"` HTTP headers to avoid downloading unmodified metadata. I checked, it is indeed used by the service, so it would be nice to use it.